packages/dbml-cli/yarn.lock

```diff
@@ -703,15 +703,6 @@
     exec-sh "^0.3.2"
```

Comment:
This file should not be touched if all you are doing is deleting package-lock.json

Reply:
reverted